scrap.py

```diff
@@ -0,0 +1,59 @@
+import datetime
+import requests
+import time
+from facebook_scraper import get_posts, exceptions
+
+"""
+    here will be the main objects and methods to 
+    scrap the facebook pages the idea is init require 
+    two arguments the page name and how many page you 
+    want to scrape
+    
+"""
+
+DELAY_SECONDS = 2
+TEST_DB = {
+    'text': 'Everything you need to know to preorder Hogwarts Legacy, the upcoming open-world RPG set in '
+            'the wizarding world of Harry Potter, hundreds of years before the hero arrived.',
+    'photo': 'https://assets-prd.ignimgs.com/2022/03/18/hogwarts-legacy-4-1647622573681.jpeg?width=1280',
+    'time': datetime.datetime(2022, 3, 24, 16, 31, 9)}
+
+
+class Post(object):
+
+    def __init__(self, page_name: str, num_pages: int):
+
+        self.page_name = page_name
+        self.num_pages = num_pages
+        self.post_info = {}
+        self.url = "https://www.facebook.com/"
+
+    def check_connection(self):
+        try:
+            get = requests.get(self.url)
+            if get.status_code == 200:
+                print(f"{self.url}: is reachable")
+            else:
+                print(f"{self.url}: is Not reachable, status_code: {get.status_code}")
+        except requests.exceptions.RequestException as e:
+            raise ConnectionError(f"{self.url}: is Not reachable \nErr: {e}")
+
+    # it's static function now later with database I will make it related to this class
+    def where_to_start(self, my_dict):
+        test_dict = TEST_DB
+        if my_dict['time'] > test_dict['time']:
+            return True
+        else:
+            return False
```

Comment:
I don't understand this mechanism how can the function know from where it starts if its only returning true or false?